mrp_production_byproduct_cost_share/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2023 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
Wrong license link. And on other python files.

Reply:
True, sorry. I am also adding the change in the cost calculation in the BOM Structure Report. When I have it done I will push with this fixed as well.